README.md

```diff
@@ -82,7 +85,7 @@ with the required permissions enabled on it.
 # Scenarios
 
 - [Create a release](#create-a-release)
-- [Create a release without a CHANGELOG.md file](#create-a-release-without-changelog)
+- [Create a release without a CHANGELOG.md file](#create-a-release-without-a-changelogmd-file)
```

Comment:
Good catch, thank you for your contribution!

```suggestion
- [Create a release without a CHANGELOG.md file](#create-a-release-without-a-changelog-file)
```

Spell check fix 😊.

Reply:
In order to make the link work after a spell check fix I had to change heading of the section as well, hope it won't make it less meaningful